packages/shared/src/components/squads/Members/PrivilegedMemberItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement } from 'react';
+import { ProfilePicture } from '../../ProfilePicture';
+import SquadMemberBadge from '../SquadMemberBadge';
+import { SourceMember } from '../../../graphql/sources';
+import { ProfileTooltip } from '../../profile/ProfileTooltip';
+import { ProfileLink } from '../../profile/ProfileLink';
+
+interface PrivilegedMemberItemProps {
+  member: SourceMember;
+}
+
+export function PrivilegedMemberItem({
+  member: { user, role },
+}: PrivilegedMemberItemProps): ReactElement {
+  return (
+    <ProfileTooltip user={user} tooltip={{ placement: 'bottom' }}>
+      <ProfileLink
+        href={user.permalink}
+        className="flex flex-row items-center rounded-10 border border-border-subtlest-tertiary p-2"
+      >
+        <ProfilePicture user={user} size="large" />
+        <div className="flex-col">
+          <span className="ml-2.5 flex overflow-hidden text-ellipsis whitespace-nowrap text-text-tertiary typo-subhead">
```

Comment:
Just a check, @omBratteng didn't you add a shorthand className for this? *(the truncating)

Reply:
oh yes, I think I remember that as well

Reply:
Ah yes, there is also a native truncate class iirc.